barretenberg/ts/src/barretenberg_wasm/barretenberg_wasm_main/index.ts

```diff
@@ -32,7 +32,7 @@ export class BarretenbergWasmMain extends BarretenbergWasmBase {
     module: WebAssembly.Module,
     threads = Math.min(getNumCpu(), BarretenbergWasmMain.MAX_THREADS),
     logger: (msg: string) => void = debug,
-    initial = 30,
+    initial = 31,
```

Comment:
why?

Reply:
it literally told me initial couldn't be 30 and needed 31... I guess it just needs more initial memory